modules/odc-interacting-with-applications-and-components.adoc

```diff
@@ -18,3 +18,13 @@ This feature is available only when you create applications using the *From Git*
 * Hover your cursor over the lower left icon on the Pod to see the name of the latest build and its status. The status of the application build is indicated as *New* (image:odc_build_new.png[title="New Build"]), *Pending* (image:odc_build_pending.png[title="Pending Build"]), *Running* (image:odc_build_running.png[title="Running Build"]), *Completed* (image:odc_build_completed.png[title="Completed Build"]), *Failed* (image:odc_build_failed.png[title="Failed Build"]), and *Canceled* (image:odc_build_canceled.png[title="Canceled Build"]).
 * Use the *Shortcuts* menu listed on the upper-right of the screen to navigate components in the *Topology* view.
 * Use the *List View* icon to see a list of all your applications and use the *Topology View* icon to switch back to the *Topology* view.
+* Use the *Find by name* field to select the components with component names that match the query. Search results may appear outside of the visible area, click *Fit to Screen* from the lower-left toolbar to resize the *Topology* to show all components.
```

Comment:
s/ click *Fit to Screen* from the lower-left toolbar to resize the *Topology* / click the *Fit to Screen* icon to resize the *Topology* view
I am not sure I understand the location specified here, I thought it was on the right. But more importantly since this is in addition to the first field I don't think we need to mention the location here. It should ideally be mentioned for the *Find by name* field, but on line 19 we mention the location, unless the line of vision has changed drastically IMO we do not need to mention it.

Reply:
this has nothing to do with shortcuts, resolved

Reply:
@pmacko1, click the Fit to Screen icon is still a valid comment. I don't think this is resolved.

Reply:
done, overlooked